lib/private/AppFramework/DependencyInjection/DIContainer.php

```diff
@@ -79,6 +79,7 @@
  * @deprecated 20.0.0
  */
 class DIContainer extends SimpleContainer implements IAppContainer {
+	private $appName;
```

Comment:
```suggestion
	private string $appName;
```
Or if it may be null in some cases, at least add a phpdoc type of `?string`